lib/tokenize.js

```diff
@@ -1,14 +1,15 @@
 /*eslint no-useless-escape: "off"*/
 
 /**
- * Remove special characters and return an array of tokens (words).
- * @param  {string} input Input string
- * @return {array}        Array of tokens
+ * Removes special characters and return an array of tokens (words).
+ * @param  {string} input Input string.
+ * @return {array}       Array of tokens.
  */
-module.exports = function(input) {
-    return input
+const tokenize = input =>
+    input
         .toLowerCase()
         .replace(/\n/g, ' ')
         .replace(/[.,\/#!$%\^&\*;:{}=_`\"~()]/g, '')
         .split(' ');
-};
+
+module.exports = tokenize;
```

Comment:
I'm not sure about this change. Why move the `module.exports` call down here?

Reply:
If there is only one exported function I think its fine to have it the assignment made along with the function declaration up top. That being said, `lib/index.js` should probably be done in the same way.  @thisandagain should probably clarify.

Reply:
@thisandagain I think this just needs one more review before its ready to be merged.